cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -244,6 +244,7 @@ function hack_csv() {
     # rook-ceph-osd --> serviceAccountName
     #     rook-ceph-osd --> rule
 
+    sed -i 's/rook-ceph-global-rules/rook-ceph-system/' "$DESIRED_CSV_FILE_NAME"
```

Comment:
I see the bad line in the csv generated is this: 
```
serviceAccountName: rook-ceph-global
```
So we need this:
```suggestion
    sed -i 's/rook-ceph-global/rook-ceph-system/' "$DESIRED_CSV_FILE_NAME"
```